beacon-chain/state/stategen/replay.go

```diff
@@ -30,32 +29,18 @@ func (s *State) ReplayBlocks(ctx context.Context, state *stateTrie.BeaconState,
 			if state.Slot() >= signed[i].Block.Slot {
 				continue
 			}
-			if featureconfig.Get().EnableStateGenSigVerify {
-				state, err = transition.ExecuteStateTransition(ctx, state, signed[i])
-				if err != nil {
-					return nil, err
-				}
-			} else {
-				state, err = executeStateTransitionStateGen(ctx, state, signed[i])
-				if err != nil {
-					return nil, err
-				}
+			state, err = executeStateTransitionStateGen(ctx, state, signed[i])
+			if err != nil {
+				return nil, err
```

Comment:
Discussed offline https://discordapp.com/channels/476244492043812875/483017808658169866/765740051450691604

This is OK to remove, the flag was never used and should have been in the form of DisableStateGenSigVerify. It was a safety net in case the state gen feature was not working properly.